src/theory/arith/nl/nl_model.cpp

```diff
@@ -1087,13 +1115,23 @@ bool NlModel::simpleCheckModelMsum(const std::map<Node, Node>& msum, bool pol)
         }
         else
         {
-          Trace("nl-ext-cms-debug") << std::endl;
-          Trace("nl-ext-cms")
-              << "  failed due to unknown bound for " << vc << std::endl;
-          // should either assign a model bound or eliminate the variable
-          // via substitution
-          Assert(false);
-          return false;
+          auto wit = d_check_model_witnesses.find(vc);
+          if (wit != d_check_model_witnesses.end())
+          {
+            // TODO(Gereon): Should do something here.
```

Comment:
Ditto.

Reply:
Not worrying and not doing anything sounds good to me.